test/functional/tests/stats/test_display_statistics.py

```diff
@@ -0,0 +1,420 @@
+    usage_stats, request_stats, block_stats_core, block_stats_cache, error_stats
+)
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_utils.size import Size, Unit
+
+
+# One cache instance per every cache mode:
+caches_count = len(CacheMode)
+cores_per_cache = 4
+# Time to wait after fio (in seconds):
+time_to_wait = 30
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_cache_config_stats():
+    """
+        title: Test CAS configuration informations for cache device.
+        description: |
+          Check CAS ability to display proper configuration informations
+          in statistics for cache device in every cache mode before and after IO.
+        pass_criteria:
+          - Cache configuration statistics matches cache properties.
+    """
+
+    with TestRun.step("Partition cache and core devices"):
+        cache_dev, core_dev = storage_prepare()
+
+    with TestRun.step(
+        f"Start {caches_count} caches (one for every cache mode) "
+        f"and add {cores_per_cache} cores per cache"
+    ):
+        caches, cores = cache_prepare(cache_dev, core_dev)
+
+    with TestRun.step(f"Get configuration statistics for each cache and validate it"):
+        validate_cache_config_statistics(caches)
+
+    with TestRun.step("Run 'fio'"):
+        fio = fio_prepare()
+        for i in range(caches_count):
+            for j in range(cores_per_cache):
+                fio.add_job().target(cores[i][j].system_path)
+        fio_pid = fio.run_in_background()
+
+    with TestRun.step(f"Wait {time_to_wait} seconds"):
+        time.sleep(time_to_wait)
+
+    with TestRun.step("Check cache configuration statistics after IO"):
+        validate_cache_config_statistics(caches)
+        caches_stats = [
+            caches[i].get_statistics(stat_filter=[StatsFilter.conf])
+            for i in range(caches_count)
+        ]
+        failed_stats = ""
+        for i in range(caches_count):
+            cache_mode = CacheMode[caches_stats[i].config_stats.write_policy.upper()]
+            if CacheModeTrait.LazyFlush in CacheMode.get_traits(cache_mode):
+                if caches_stats[i].config_stats.dirty_for.total_seconds() <= 0:
+                    failed_stats += (
+                        f"For cache nr {caches[i].cache_id} in {cache_mode} cache mode, value is "
+                        f"{caches_stats[i].config_stats.dirty_for}, should be greater then 0\n")
+            else:
+                if caches_stats[i].config_stats.dirty_for.total_seconds() != 0:
+                    failed_stats += (
+                        f"For cache nr {caches[i].cache_id} in {cache_mode} cache mode, value is "
+                        f"{caches_stats[i].config_stats.dirty_for}, should equal 0\n")
```

Comment:
If you would extend `validate_cache_config_statistics()` interface with `after_io` param, this check could be moved to `validate_cache_config_statistics()`

Reply:
Done